docs-src/guide/02-writing-templates.md

```diff
@@ -16,6 +16,26 @@ html`<h1>Hello World</h1>`
 html`<h1>Hello ${name}</h1>`
 ```
 
+## Rendering Templates
+
+A lit-html template expresion does not cause any DOM to be created or updated. It's only a description of DOM, called a TemplateResult. To actually create or update DOM, you need to pass the TemplateResult to the `render()` function, along with a container to render to:
+
+```ts
+import {html, render} from 'lit-html';
+
+const sayHi = (name) => html`<h1>Hello ${name}</h1>`;
+render(sayHi('Amy'), document.body);
+
+// subsequent renders wiill update the DOM
+render(sayHi('Zoe'), document.body);
+```
+
+### Render Options
+
+`render()` also takes an options argument that allows you to specify:
```

Comment:
@justinfagnani Add a note that render options should not change between calls to render?  Or at least that some options may only be respected on the first render?

Reply:
done